hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/records/SnapshotModeOp.java

```diff
@@ -433,6 +434,18 @@ private void fuzzyMatch(
                                 + expectedMessage + " and " + actualMessage + " - " + mismatchContext.get());
             }
 
+            final boolean isFileUpdate = FileUpdateTransactionBody.class.isAssignableFrom(expectedType);
+            final boolean isForFile121 =
+                    FileID.newBuilder().setFileNum(121).build().equals(expectedField.getValue());
+            final boolean isForFile111 =
+                    FileID.newBuilder().setFileNum(111).build().equals(expectedField.getValue());
+            // skip matching FileUpdate for properties file 121 or fee schedules file 111 since depending on if leaky
+            // tests are run before or not,
+            // the FileUpdate contents can be different
+            if (isFileUpdate && (isForFile121 || isForFile111)) {
```

Comment:
Can't we just updated `ParsedItem.isPropertyOverride()` [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/domain/ParsedItem.java#L55) to also ignore fee schedule updates?

Actually, should we generalize that method to `isSpecialFileChange()` and do something like,
```
        return itemRecord.getReceipt().getStatus() == SUCCESS
                && (itemBody.hasFileUpdate() || itemBody.hasFileAppend())
                && PROPERTIES_FILE_ID.equals(itemBody.getFileUpdate().getFileID()) || FEE_SCHEDULE_FILE_ID.equals(itemBody.getFileUpdate().getFileID());

```

Reply:
Thanks Michael, I added it [as such](https://github.com/hashgraph/hedera-services/pull/10631/commits/cec1288baaabebe8c06520e55eace57c9dd3ae22#diff-fa1ee390421aaf33ca4cb842c66f37462f4f660326f6c3c8ed7ef575f1410101R55-R70).